components/benefit_markets/app/models/benefit_markets/locations/us_county.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module BenefitMarkets
+  module Locations
+    class UsCounty
```

Comment:
This model looks similar to [CountyZip Model ](https://github.com/ideacrew/enroll/blob/trunk/components/benefit_markets/app/models/benefit_markets/locations/county_zip.rb#L7-7 ), what do you think of adding the `fips_code` field CountyZip model

Reply:
We can place counties code over here but currently, in Enroll we are using the countyzip's model to load counties and zips only for a particular state, but for our case we need to load all the Us counties.